crates/script/src/lib.rs

```diff
@@ -210,6 +213,9 @@ pub struct ScriptArgs {
 
     #[command(flatten)]
     pub retry: RetryArgs,
+
+    #[clap(flatten)]
+    pub shell: ShellOpts,
```

Comment:
Renaming `ShellOpts` to `GlobalOpts` + consistent enforcement will be handled in https://github.com/foundry-rs/foundry/issues/8408

It will be possible to access the global shell options either directly by e.g. `shell::is_quiet`, etc.. as well as by `args.global.shell().is_quiet()`